### PR TITLE
Chia Seed Pudding Recipe

### PR DESCRIPTION
**Pull request by johnaeastman**
# Chia Seed Pudding Recipe 🥣

## Ingredients:
- Black chia seeds
- Almond milk or other milk
- Honey
- Vanilla extract
- Select toppings:
  - [Blueberries]
  - [Almonds]
  - [Granola]
  - [Peaches]

## Equipment needed:
- Large mason jar

## Steps for preparation:
1. Pour ingredients in a Mason jar in the following proportions (will be about 3x for large Mason jar):
  - 8 tablespoons chia seeds
  - 2 cups almond milk or milk of choice
  - 4 tsp honey or other sweetener
  - 1 tsp vanilla extract, more to taste
2. Mix well. Let settle for 2-3 minutes and then mix again very well until you see NO CLUMPING.
3. Cover the jar and store in fridge overnight or for at least 2 hours.
  - Note: If mixture is too liquidy at any point, add more chia seeds in 1 tsp increments.
5. When ready, top with your favorite toppings and honey, and enjoy!

### Notes:
Easy and delicious breakfast!

### Codemates #
@kongtianshou 
@tfantis
@chwong11
@lsallersonberk

### CODE REVIEW

**Comment by johnaeastman** [comment]
Please add your toppings above! Thanks! @caroline-tsoi @tfantis @chwong11 @LSallersonberk

File: breakfast/readme.md [hunk 1]
```diff
@@ -1,21 +1,35 @@
...
 ## Ingredients:
-
+- Black chia seeds
+- Almond milk or other milk
+- Honey
+- Vanilla extract
+- Select toppings:
+  - [Strawberries]
+  - [Kiwi]
+  - [❗️ADD❗️]
```
Comment (tfantis) [inline breakfast/readme.md:15]: blueberries

File: breakfast/readme.md [hunk 2]
```diff
@@ -1,21 +1,36 @@
...
-## Time to prepare: 
+## Time to prepare: 10 minutes (refrigerate overnight or for at least 2 hours)
 
 ## Ingredients:
-
+- Black chia seeds
+- Almond milk or other milk
+- Honey
+- Vanilla extract
+- Sugar
```
Comment (chwong11) [inline breakfast/readme.md:12]: Honey and sugar might be a bit extra, but lets go with it!

File: breakfast/readme.md [hunk 3]
```diff
@@ -1,21 +1,36 @@
...
+- Select toppings:
+  - [Strawberries]
+  - [Kiwi]
+  - [❗️ADD❗️]
```
Comment (chwong11) [inline breakfast/readme.md]: @tfantis 
Can you add one more ingredient of your liking? Thanks!

**Review by chwong11 — APPROVED** [review]
I like this breakfast the way I like my traveling...quick and easy! Just need one more ingredient commit from @tfantis and we can approve this PR.

**Review by kongtianshou — APPROVED** [review]

**Pull request merged**